fastaparse.py

```diff
@@ -0,0 +1,179 @@
+import re
+
+mass_table = {
+    "A": 89.047678,
+    "C": 121.019749,
+    "D": 133.037508,
+    "E": 147.053158,
+    "F": 165.078979,
+    "G": 75.032028,
+    "H": 155.069477,
+    "I": 131.094629,
+    "K": 146.105528,
+    "L": 131.094629,
+    "M": 149.051049,
+    "N": 132.053492,
+    "O": 255.158292,
+    "P": 115.063329,
+    "Q": 146.069142,
+    "R": 174.111676,
+    "S": 105.042593,
+    "T": 119.058243,
+    "U": 168.964203,
+    "V": 117.078979,
+    "W": 204.089878,
+    "Y": 181.073893,
+}
+
+codons = {
+    "UUU": "F",
+    "CUU": "L",
+    "AUU": "I",
+    "GUU": "V",
+    "UUC": "F",
+    "CUC": "L",
+    "AUC": "I",
+    "GUC": "V",
+    "UUA": "L",
+    "CUA": "L",
+    "AUA": "I",
+    "GUA": "V",
+    "UUG": "L",
+    "CUG": "L",
+    "AUG": "M",
+    "GUG": "V",
+    "UCU": "S",
+    "CCU": "P",
+    "ACU": "T",
+    "GCU": "A",
+    "UCC": "S",
+    "CCC": "P",
+    "ACC": "T",
+    "GCC": "A",
+    "UCA": "S",
+    "CCA": "P",
+    "ACA": "T",
+    "GCA": "A",
+    "UCG": "S",
+    "CCG": "P",
+    "ACG": "T",
+    "GCG": "A",
+    "UAU": "Y",
+    "CAU": "H",
+    "AAU": "N",
+    "GAU": "D",
+    "UAC": "Y",
+    "CAC": "H",
+    "AAC": "N",
+    "GAC": "D",
+    "UAA": "Stop",
+    "CAA": "Q",
+    "AAA": "K",
+    "GAA": "E",
+    "UAG": "Stop",
+    "CAG": "Q",
+    "AAG": "K",
+    "GAG": "E",
+    "UGU": "C",
+    "CGU": "R",
+    "AGU": "S",
+    "GGU": "G",
+    "UGC": "C",
+    "CGC": "R",
+    "AGC": "S",
+    "GGC": "G",
+    "UGA": "Stop",
+    "CGA": "R",
+    "AGA": "R",
+    "GGA": "G",
+    "UGG": "W",
+    "CGG": "R",
+    "AGG": "R",
+    "GGG": "G"
+}
+
+
+def parse(path):
+    """
+    Function parses a fasta file and returns a dictionary with seq_ids and sequences
+    :param path: path to fasta file
+    :return: dictionary with seq_id as keys and sequence for seq_id as values
+    """
+    with open(path, 'r') as fasta:
+        result = dict()
+        for idx, line in enumerate(fasta):
```

Comment:
Since you don’t employ idx anywhere else, you probably can receive the same results with just:
```suggestion
        for line in fasta:
```
However, it is also possible to use this part further if you rewrite the body of the loop, but I like this version, since  it's human-readable and nice :)

Reply:
I forgot to remove _idx_ , thanks :)